### DISCUSSION TITLE
How to use the bypass (see previous discussion)

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#135] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://user-images.githubusercontent.com/77510164/197800199-f130a4e7-8c71-4852-bb46-e77cc7258ac5.mp4

**Comment by Sxri** [comment]
Twitter is automatically unblocked for us since teachers use it.

**Reply by ghost** [reply]
lol that's concerning

**Comment by skoopy-star** [comment]
task manager is blocked

**Reply by ghost** [reply]
ok, I'm working on a workaround

**Reply by skoopy-star** [reply]
ty

**Comment by Rainzy54** [comment]
how do you open that page in the begining

**Reply by ghost** [reply]
I crash goguardian with search+esc> end task

**Reply by Rainzy54** [reply]
i cant end it

**Reply by ghost** [reply]
like I said in like every other comment, I can't help you right now, that's not my problem

**Comment by SomePVZPlayerMatt** [comment]
the blocker that my school district uses is contentkeeper, and it sucks. just look at the extension reviews https://chrome.google.com/webstore/detail/ckauthenticator/jdogphakondfdmcanpapfahkdomaicfa
also my school blocked end process for task manager

**Reply by ghost** [reply]
yeah I can't help right now, read the 2 billion other comments I said that on
